documentation/QUICKSTART.md

```diff
@@ -2,135 +2,116 @@
-export CURRENT_TIME
-CURRENT_TIME=$(date +%s)
-export TRACKER_RUN_NAME="${CURRENT_TIME}"
-export VALIDATION_PROMPT="a studio portrait photograph of a teddy bear holding a sign that reads, 'Hello World'"
-export VALIDATION_GUIDANCE=5.5
-export VALIDATION_STEPS=100
-
-
-# Location of training data.
-export BASE_DIR="/path/to/outputs"
-export DATALOADER_CONFIG="multidatabackend.json"
-export INSTANCE_DIR="${BASE_DIR}/datasets/training_data"
-export OUTPUT_DIR="${BASE_DIR}/models"
-
-export PUSH_TO_HUB="false"
-export PUSH_CHECKPOINTS="false"
-# This is how many checkpoints we will keep. Two is safe, but three is safer.
-export CHECKPOINTING_LIMIT=10
-
-# By default, images will be resized so their SMALLER EDGE is 1024 pixels, maintaining aspect ratio.
-# Setting this value to 768px might result in more reasonable training data sizes for SDXL.
-export RESOLUTION=1
-export MINIMUM_RESOLUTION=$RESOLUTION
-export VALIDATION_RESOLUTION=1280x768
-export VALIDATION_NUM_INFERENCE_STEPS=50
-export RESOLUTION_TYPE="area"
-export ASPECT_BUCKET_ROUNDING=2
-
-# Adjust this for your GPU memory size. This, and resolution, are the biggest VRAM killers.
-export TRAIN_BATCH_SIZE=1
-export GRADIENT_ACCUMULATION_STEPS=1
-export MAX_NUM_STEPS=1000
-export NUM_EPOCHS=0
-
-# Set this to 'true' if you are using a patched Diffusers install.
-# Ignore this if you don't know what it does. Having it at 'false' uses more VRAM.
-export USE_GRADIENT_CHECKPOINTING=false
-
-export LR_SCHEDULE="constant"
-export LR_WARMUP_STEPS=0
-export CAPTION_DROPOUT_PROBABILITY=0.1
-export OPTIMIZER="adamw_bf16"
-
-
-# Reproducible training. Set to -1 to disable.
-export TRAINING_SEED=42
-export VALIDATION_SEED=2
-export MIXED_PRECISION="bf16"
-
-# This has to be changed if you're training with multiple GPUs.
-export TRAINING_NUM_PROCESSES=1
-
-# Leave this alone.
-export TRAINER_EXTRA_ARGS=""
-export ACCELERATE_EXTRA_ARGS=""
-export USE_XFORMERS=false
+# MacOS
+poetry install --no-root -C install/apple
+
+# Linux
+poetry install --no-root
+
+# Linux with ROCM
+poetry install --no-root -C install/rocm
 ```
 
-#### Dataloader
+### Setting up the environment
+
+To run SimpleTuner, you will need to set up a configuration file, the dataset and model directories, and a dataloader configuration file.
+
+#### Configuration file
 
-Place the following in `SimpleTuner/multidatabackend.json`
+Copy `sdxl-env.sh.example` to `sdxl-env.sh`:
+
+```bash
+cp sdxl-env.sh.example sdxl-env.sh
+```
+
+There, you will need to modify the following variables:
+
+- `MODEL_TYPE` - Set this to `lora`.
+- `STABLE_DIFFUSION_3` - Set this to `true`.
+- `MODEL_NAME` - Set this to `stabilityai/stable-diffusion-3-medium-diffusers`. Note that you will need to log in to Huggingface and be granted access to download this model. We will go over logging in to Huggingface later in this tutorial.
+- `BASE_DIR` - Set this to the directory where you want to store your outputs and datasets. It's recommended to use a full path here.
```

Comment:
there were a couple other important keys like bitfit=false, dora=false

Reply:
In `sdxl-env.sh.example` `USE_DORA` is already `false` and `USE_BITFIT` is automatically set to `false`

Reply:
it actually has a conditional statement in there and it enables bitfit by default for full model training.

but i guess your guide is for lora, so this is ok